src/pint/models/jump.py

```diff
@@ -242,3 +242,32 @@ def add_jump_and_flags(self, toa_table):
             dict1["jump"] = str(ind)
             dict1["gui_jump"] = str(ind)
         return name
+
+    def delete_not_all_jump_toas(self, toa_table, jump_num):
+        """Unjumps a subset of jumped TOAs when the selected subset  doesn't exactly
+        overlap with all the TOAs in a given jump.
+
+        Helper function for pintk. Mailny used when one wants to delete the jump from
+        a subset of jumped TOAs.
+
+        Parameters
+        ----------
+        toa_table: list object
```

Comment:
is that the correct object type?  is it not a `astropy.table.Table`?

Reply:
That's a bad naming from my side. The object refers to the list of flags of the selected TOAs from the TOA table. I'll change the name...